src/scraper/__init__.py

```diff
@@ -149,31 +298,76 @@ def scrape_main_page(self, base_url, odyssey_version, session, notes, logger, ms
                 ms_wait=ms_wait,
             )
             main_soup = BeautifulSoup(main_page_html, "html.parser")
-            return main_page_html, main_soup
+        except Exception as e:
```

Comment:
Probably catch a requests specific error. [requests.exceptions.RequestException](https://requests.readthedocs.io/en/latest/api/#requests.RequestException) is the base type for all exceptions that library can generate

Reply:
I need to think through what each of the different potential errors might be for each of these different methods, but this may be a ticket I create for later.